install/helm/openchoreo-control-plane/templates/thunder/bootstrap-scripts-configmap.yaml

```diff
@@ -0,0 +1,227 @@
+{{- if .Values.thunder.enabled }}
+{{- if .Values.thunder.bootstrap.enabled }}
+{{- $scheme := ternary "http" "https" .Values.thunder.configuration.server.httpOnly }}
+{{- $curlInsecure := ternary "" "-k" .Values.thunder.configuration.server.httpOnly }}
+{{- $port := .Values.thunder.configuration.server.port }}
+{{- $thunderUrl := printf "%s://localhost:%d" $scheme (int $port) }}
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: openchoreo-thunder-bootstrap
+  namespace: {{ .Release.Namespace }}
+  labels:
+    {{- include "openchoreo-control-plane.labels" . | nindent 4 }}
+    app.kubernetes.io/component: thunder-bootstrap
+  annotations:
+    "helm.sh/hook": pre-install
```

Comment:
how does this work if the user update the base url? like at the first install they used something like `thunder.example.com` and later changed to `idp.example.com`?